main.cpp

```diff
@@ -311,8 +341,15 @@ KeithEmerson::~KeithEmerson()
 
 void KeithEmerson::activateKeyboardGodMode()
 {
-    for(auto& keyboard: keyboards) FIXME
-        keyboard.playChord();   
+    for(auto& keyboard: keyboards)
+    {
+        keyboard.playChord(); 
+    }  
+}
+
+SynthMoog& KeithEmerson::getKeyboard(unsigned int whichKeyboard)
+{
+    return this->keyboards[whichKeyboard];
```

Comment:
implement bounds checking.   if you have 15 keyboards and pass in `15` for `whichKeyboard`, will this code work or will it crash?

This is why I don't typically let people use arrays or containers in these early projects before the relevant topics have been covered.
- [ ] revise